CHANGES.md

```diff
@@ -1,9 +1,15 @@
 ## 0.1.1
 
-* New `thriftProtocol` config option to the thrift protocol to be
-  specified.
+* New `thriftProtocol` config option allows the thrift protocol to be
+  specified. We currently support `binary` (default) and `compact`.
+* Overhaul to the internals of the configuration system. Note that this
+  introduces a *breaking change*: we no longer support `router` options
+  configured globally at the root of the configuration file.
```

Comment:
I would leave out the overhaul to the internals, since it's not really user-facing, and talk about the motivation for the breaking config change instead.